<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="bt_load">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Load PLY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_save">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save PLY</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_filename">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>251</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_inputsizewh">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>40</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Input size W*H:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_w">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_h">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_xmultiple">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_filter">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter points</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_filter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0.01</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_alignmajor">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Align major</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pointsize">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>65</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Point size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_filteredpointsize">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>65</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_resetcam">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>180</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Camera View</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_viewup">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_viewdown">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_viewleft">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_camera">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera View</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_viewright">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_alignmiddle">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Align middle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_alignminor">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>220</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Align minor</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_major0">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>major0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_major1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>major1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_major2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>major2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_middle1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>280</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>middle1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_middle0">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>260</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>middle0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_middle2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>middle2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_minor1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>280</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>minor1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_minor0">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>260</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>minor0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_minor2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>300</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>minor2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_boundingbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Bounding Box</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_x_rot">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>320</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_y_rot">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>320</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_z_rot">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>320</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_voxel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>37</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply Voxel</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_xrot">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>320</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>X-rot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_yrot">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>320</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Y-rot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_zrot">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>320</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Z-rot</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_bb_xsize">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>x size</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_bb_zsize">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>z size</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_bb_ysize">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>y size</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_bbx">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>x:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_bby">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_bbz">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>z:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_pca">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>PCA calculate</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>442</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
